frontend/src/components/SignOutButton.tsx

```diff
@@ -18,8 +17,6 @@ export const SignOutButton = () => {
   return (
     <Button
       variant="outlined"
-      size="large"
-      color="secondary"
       className="ml-auto"
       onClick={() => handleLogout(instance)}
     >
```

Comment:
Just changed this to match the sign in button. I know we're going to override and move it elsewhere but just want to make sure it's still visible for debugging purposes.